core/src/main/java/fr/sncf/osrd/envelope_sim/PhysicsRollingStock.java

```diff
@@ -56,4 +56,8 @@ static double getMaxEffort(double speed, TractiveEffortPoint[] tractiveEffortCur
     /** The maximum acceleration, in m/s^2, which can be applied at a given speed, in m/s */
     record TractiveEffortPoint(double speed, double maxEffort) {
     }
+
+    record CurvesAndConditions(RangeMap<Double, TractiveEffortPoint[]> curves,
+                               RangeMap<Double, PhysicsPath.ModeAndProfile> conditions) {
```

Comment:
I just don't get it, aren't conditions used to get the curves? is it just to debug / display?